src/Propel/Generator/Command/InitCommand.php

```diff
@@ -264,7 +264,7 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
         $outputDir = sys_get_temp_dir();
 
         $this->getApplication()->setAutoExit(false);
-        $fullDsn = sprintf('%s;user=%s;password=%s', $options['dsn'], urlencode($options['user']), urlencode($options['password']));
+        $fullDsn = sprintf('%s;user=%s;password=%s', $options['dsn'], $options['user'], $options['password']);
```

Comment:
@staabm I was unable to find any encoding scheme that works with MySQL authentication.